.github/workflows/app-e2e.yml

```diff
@@ -2,10 +2,10 @@ name: BE App E2E tests
 
 on:
   workflow_call:
-    secrets:
-      ALLURE_TOKEN:
-        description: 'A token passed from the caller workflow'
-        required: true
+    # secrets:
+    #   ALLURE_TOKEN:
+    #     description: 'A token passed from the caller workflow'
+    #     required: true
```

Comment:
Allure reports were disabled for the time being, so this variable is not used. However, since it's required, the workflow fails for forked PRs.